Add snapshot modes (#974)
home-assistant/developers.home-assistant: docs/add-ons/configuration.md

| panel_icon | string | no | Default: `mdi:puzzle`. MDI icon for the menu panel integration.
| panel_title | string | no | Default is the add-on name, but can be modified with this option.
| panel_admin | bool | no | Default `true`. Make menu entry only available with admin privileged.

> | snapshot | string | no | `hot` (default) or `cold`. If `cold`, the supervisor turns the add-on off before taking a snapshot (the `pre/post` options are when used).

Reviewer: ```suggestion
| snapshot | string | no | `hot` (default) or `cold`. If `cold`, the supervisor turns the add-on off before taking a snapshot (the `pre/post` options are ignored when `cold` is used).
```

> | snapshot | string | no | `hot` (default) or `cold`. If `cold`, the supervisor turn the add-on off before he take an snapshot and the options for `pre/post` are not working.

Reviewer: ```suggestion
| snapshot | string | no | `hot` (default) or `cold`. If `cold`, the supervisor turns the add-on off before taking a snapshot (the `pre/post` options are when used).
```
